### PR TITLE
Add CVE-2021-39181 for GHSA-596v-3gwh-2m9w

### PR DESCRIPTION
**Pull request by shelbyc**
Add CVE-2021-39181 for GHSA-596v-3gwh-2m9w

### CODE REVIEW

File: 2021/39xxx/CVE-2021-39181.json [hunk 1]
```diff
@@ -1,18 +1,96 @@
 {
```
Comment (cve-team) [inline 2021/39xxx/CVE-2021-39181.json:1]: 'https://jira.openolat.org/browse/OO-5548 ' does not match '^(ftp|http)s?://\\S+$'

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2021/39xxx/CVE-2021-39181.json (hunk 1 above)
Comment (shelbyc) [reply 2021/39xxx/CVE-2021-39181.json:1]: Removed unnecessary space at the end of the URI

Comment (shelbyc) [reply 2021/39xxx/CVE-2021-39181.json:1]: Removed space from the end of the URI

**Pull request merged**